packages/base/src/dialogs/symbology/symbologyUtils.ts

```diff
@@ -94,33 +100,19 @@ export namespace Utils {
     selectedRamp: string,
     nClasses: number,
   ) => {
-    let colorMap = colormap({
+    const colorMap = colormap({
       colormap: selectedRamp,
-      nshades: nClasses > 9 ? nClasses : 9,
+      nshades: 256,
```

Comment:
```suggestion
      nshades: nClasses > 9 ? nClasses : 9,
```
This should be controlled by the caller. What's the purpose of this change?